### DISCUSSION TITLE
Sample project

### DISCUSSION THREAD

**Discussion by eric-ampire** [GitLiveApp/firebase-kotlin-sdk#239] [Q&A]
Does anyone have an open source project that they developed with the sdk, if possible can they share the links.

**Comment by solvek** [comment]
Strange... why no answer to this question?

**Comment by carlosgub** [comment]
I develop a really MVP only showing some values from firestore that I create. https://github.com/carlosgub/Firestore-example-kmm

**Comment by Hanif-MI** [comment]
Has anyone implemented phone authentication? or any sample project available?
